en/docs/install-and-setup/setup/sso/okta-as-an-external-idp-using-oidc.md

```diff
@@ -183,28 +191,37 @@ In the following document we will explain how to connect OKTA as a third party I
     </tbody>
 </table>
 
-5. Expand Claim configuration -> Basic claim configuration and add the following claim configurations
+5. Expand **Claim Configuration** -> **Basic Claim Configuration**. Add the claim configurations as shown in the image below.
     [![]({{base_path}}/assets/img/learn/okta-apim-idp-claims-details.png)]({{base_path}}/assets/img/learn/okta-apim-idp-claims-details.png) 
 
-6. Expand Role configuration and add the following role. Here we check if the user that is being logged in has the role `any` and assign him the local role okta_role
+6. Expand **Role configuration** and add `okta_role` as shown below. You can check if the user logged in has the role `any` and assign the local `okta_role`.
 
     <img src="{{base_path}}/assets/img/learn/okta-apim-role-oidc-role-mapping.png"/>
 
-7. Enable Just in time provisioning so that the user will be saved in the API-Manager user store
+7. Enable **Just-in-Time Provisioning** for the user to be saved in the API Manager user store.
 
     <img src="{{base_path}}/assets/img/learn/okta-apim-role-oidc-jit.png"/>
 
-8. Navigate to Service providers -> list as shown below. There are two service providers created apim_publisher, apim_devportal. Click on edit on apim_publisher.
+    !!!info
+        When Just-In-Time Provisioning is enabled, the user details will be saved in the API Manager user store. User profile details will be update via federation following each login event. To preserve the user profile details without any changes you need to enable the `SystemRolesRetainedProvisionHandler`.
```

Comment:
The above sentence, ("To preserve the user profile details without any changes you need to enable the 'SystemRolesRetainedProvisionHandler'.") needs to be corrected as follows,
"To preserve previously assigned system roles (Application/, Workflow/) in the user profile, you need to enable the 'SystemRolesRetainedProvisionHandler'."